### PR TITLE
WRQ-27436: Added public classes to expand size of title and content in Alert

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We decided to provide a public class to expand size of title and content in Alert so that full text can be shown wider on the screen.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I added public classes to Alert.

- alert
- fullscreen
- title
- content

Title and content width can be expanded using these public classes.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-27436

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Comment by seunghoh** [comment]
Our last decision was "Enact will provide a public class for apps to override the width of header and contents at once" Did we change our plan? 
Even more, `showLongTitle` sounds not clear to me.

**Comment by 0x64** [comment]
Adding public class(s) is our decision.

File: tests/screenshot/apps/components/Alert.js [hunk 1]
```diff
@@ -9,14 +9,16 @@ import Scroller from '../../../../Scroller';
 
 import img from '../../images/300x300.png';
 
+import css from './Alert.module.less';
 import {withConfig, withProps, LoremString} from './utils';
```
Comment (juwonjeong) [inline tests/screenshot/apps/components/Alert.js:13]:
```suggestion
import {withConfig, withProps, LoremString} from './utils';

import css from './Alert.module.less';
```

File: tests/screenshot/apps/components/Alert.js [hunk 2]
```diff
@@ -9,14 +9,16 @@ import Scroller from '../../../../Scroller';
...
```
Comment (juwonjeong) [inline tests/screenshot/apps/components/Alert.js:15]:
```suggestion
import {withConfig, withProps, LoremString} from './utils';

import css from './Alert.module.less';
```

File: Alert/Alert.js [hunk 3]
```diff
@@ -140,12 +156,13 @@ const AlertBase = kind({
 	},
 
 	styles: {
-		css,
-		className: 'alert'
+		css: componentCss,
+		className: 'alert',
+		publicClassNames: ['alert', 'fullscreen', 'title', 'content']
```
Comment (juwonjeong) [inline Alert/Alert.js:161]:
```suggestion
		publicClassNames: ['alert', 'content', 'fullscreen', 'title']
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `sandstone/Alert` public class names `alert`, `fullscreen`, `title`, and `content`
```
Comment (juwonjeong) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Alert` public class names `alert`, `content`, `fullscreen`, and `title`
```

File: tests/screenshot/apps/components/Alert.js (hunk 2 above)
Comment (vJIYEv) [reply tests/screenshot/apps/components/Alert.js]: ok

File: CHANGELOG.md (hunk 4 above)
Comment (vJIYEv) [reply CHANGELOG.md]: ok

File: Alert/Alert.js [hunk 5]
```diff
@@ -64,6 +64,22 @@ const AlertBase = kind({
...
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `alert` - The root class name
+		 * * `content` - The content node of the alert
+		 * * `fullscreen` - The alert node with fullscreen
+		 * * `title` - The title node of the fullscreen alert
```
Comment (juwonjeong) [inline Alert/Alert.js:76]:
```suggestion
		 * * `content` - The content component class
		 * * `fullscreen` - Applied to a `type='fullscreen'` alert
		 * * `title` - The title component class
```

Comment (vJIYEv) [reply Alert/Alert.js]: ok

**Review by juwonjeong — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/Alert.js [hunk 6]
```diff
@@ -11,12 +11,14 @@ import img from '../../images/300x300.png';
...
 
+import css from './Alert.module.less';
 
 // Only type: 'fullscreen' supports title prop
 const fullscreenTests = [
 	<Alert open title="Title" />,
 	<Alert open>Alert!</Alert>,
-	<Alert open>{LoremString}</Alert>
+	<Alert open>{LoremString}</Alert>,
+	<Alert open title="Loooooooooooooooooooooong Title with custom width" css={css}>{LoremString}</Alert>
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/Alert.js:21]:
```suggestion
	<Alert open title="Loooooooooooooooooooooong title with custom width" css={css}>{LoremString}</Alert>
```

Comment (vJIYEv) [reply tests/screenshot/apps/components/Alert.js]: ok

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**